src/app/(main)/EventPage.tsx

```diff
@@ -110,7 +126,11 @@ export const EventPage = ({ eventId }: { eventId: string }) => {
             <OutputLink label="Map" value={activity.mapId} href={`https://sartopo.com/m/${activity.mapId}`} />
           </Box>
           <Box>
-            <OutputText label="Status" value={isActivityActive ? 'In Progress' : 'Complete'} />
+            <OutputText label="Mission Status" value={getActivityStatus(activity)} />
+            <OutputText label="Active Responders" value={Object.values(activity.participants).reduce(reduceActive, 0)}></OutputText>
+            <OutputText label="Standby" value={Object.values(activity.participants).reduce(reduceStandby, 0)}></OutputText>
+            <OutputText label="Responding" value={Object.values(activity.participants).reduce(reduceSignedIn, 0)}></OutputText>

```

Comment:
Would this be better as "En Route"? Responding sounds very similar to Active Responders.

Reply:
I went with Responders because the individual status that you see when you're signed in is "Responding". I'm open to debate, just trying to align the terminology that we're using.